vulkaninfo/volk/LICENSE.md

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2018-2019 Arseny Kapoulkine
```

Comment:
The volk-adding commit should also include details on exactly which URL and version/tag/commit was brought over from volk's repo